Completion/Unix/Command/_code

```diff
@@ -0,0 +1,36 @@
+#compdef code
+
+local arguments
+
+arguments=(
+  '(-d --diff)'{-d,--diff}'[compare two files with each other]:file to compare:_files:file to compare with:_files'
+  '(-a --add)'\*{-a,--add}'[add specified directory (or directories) to the last active window.]:directory:_directories'
+  '(-g --goto)'{-g,--goto}'[open a file at the path on the specified line and character position]:file\:line[\:character]:_files -r \:'
+  '(-n --new-window -r --reuse-window)'{-n,--new-window}'[open a new window]'
+  '(-n --new-window -r --reuse-window)'{-r,--reuse-window}'[open a file or directory in the last active window]'
+  '(-w --wait)'{-w,--wait}'[wait for the files to be closed before returning]'
+  '--locale=[specify the locale to use]:locale (e.g. en-US or zh-TW):(de en en-US es fr it ja ko ru zh-CN zh-TW bg hu pt-br tr)'
+  '--user-data-dir[specify the directory that user data is in]:directory:_directories'
+  '(-v --version)'{-v,--version}'[print version]'
+  '(-h --help)'{-h,--help}'[print usage]'
+  '--extensions-dir[specify the root path for extensions]:root path:_directories'
+  '--list-extensions[list the installed extensions]'
+  '--show-versions[show versions of installed extensions, when using --list-extension]'
+  '--install-extension[specify extension to install]:id or path:_files -g "*.vsix"'
+  '--uninstall-extension[specify extension to uninstall]:id or path:_files -g "*.vsix"'
```

Comment:
This is a very pedantic point but in completion, it is common to use the glob qualifier (-.) with _files -g.
So _files -g "*.vsix(-.)"
That only matters if you have something like a directory named with the extension.

Reply:
Done.

Thanks again for your comments!